src/Analysis.R

```diff
@@ -5,24 +5,14 @@ library(forcats)
 library(stringr)
 library(tidyr)
 library(labelled)
-# library(gtsummary) ## TODO: Install {gtsummary} if necessary
-
```

Comment:
I'd rather not delete this, for leaving the option to "make and commit changes" (also, it's a good excuse to letting people know about {gtsummary} if they don't, super cool fresh tables for you paper right out of the oven! :D)